pippo-core/src/main/java/ro/pippo/core/Request.java

```diff
@@ -436,21 +464,23 @@ public String getContextPath() {
         return contextPath;
     }
 
-    public String getFilterPath() {
-        return filterPath;
+    public String getApplicationPath() {
+        return applicationPath;
     }
 
     /**
-     * Returns a relative path to the filter path and context root.
+     * Returns a request path relative to the Pippo application path.
      */
-    public String getRelativePath() {
+    public String getApplicationRequestPath() {
```

Comment:
I have the same problem with the name. I don't know if it's not similar with `getApplicationUri()`.